data-source/jsonnet/northern-ireland/blocks/individual/identity-and-health/sexual_identity.jsonnet

```diff
@@ -59,12 +56,19 @@ local proxyTitle = {
   id: 'sexual-identity',
   question_variants: [
     {
-      question: question(nonProxyTitle, nonProxyDetailAnswerLabel),
+      question: question(nonProxyTitle),
       when: [rules.proxyNo],
     },
     {
-      question: question(proxyTitle, proxyDetailAnswerLabel),
+      question: question(proxyTitle),
       when: [rules.proxyYes],
     },
   ],
+  routing_rules: [
+    {
+      goto: {
+        group: 'qualifications-group',
```

Comment:
Is this routing rule needed as it would go to this group anyway?

Reply:
It was sort of insurance against this group being moved, IIRC.

Reply:
Decided to remove. It adds a different risk, but it's the one with less code.